dev/service-registry/devmode-scaladsl/src/main/scala/com/lightbend/lagom/devmode/scaladsl/LagomDevModeComponents.scala

```diff
@@ -2,15 +2,16 @@
  * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
  */
 
-package com.lightbend.lagom.scaladsl.devmode
+package com.lightbend.lagom.devmode.scaladsl
```

Comment:
This change is breaking API and maybe not necessary (only changes order)

Reply:
Note how this change snowballed into multiple changes in the `docs/`.

Reply:
I will check and rollout back